src/utils/graph/graph.js

```diff
@@ -4,12 +4,12 @@ import { routing } from './routing';
 
 const defaultOptions = {
   layout: {
-    spaceX: 16,
+    spaceX: 14,
     spaceY: 110,
     layerSpaceY: 55,
-    basisX: 1500,
+    spreadX: 2.2,
```

Comment:
What does both the `basisX` and `spreadX` values refer to, and in this case, what does this change effectively means and do? It would help if we have some comments for the `defaultOptions` section for context.